wiki/osu!supporter/de.md

```diff
@@ -1,67 +1,55 @@
----
-outdated: true
----
-
 # osu!supporter
 
-![osu!supporter Abzeichen](img/store-product.jpg?1 "osu!supporter Abzeichen")
+*Für die osu!supporter Seite der Website, siehe: [Das Spiel unterstützen](https://osu.ppy.sh/home/support)*
+
+## Vorteile
+
+*Eine Liste der Extras, die man als osu!supporter bekommt, findest du hier: [Das Spiel unterstützen](https://osu.ppy.sh/home/support)*
+
+Im Allgemeinen unterstützt du durch den Kauf des osu!supporter-Tags direkt die Entwicklung des Spiels und erhältst gleichzeitig einige zusätzliche Extras. Es muss beachtet werden, dass die zusätzlichen Funktionen von osu!supporter niemals das Gameplay, die Performance oder deinen Score in irgendeiner Weise zu deinem Vorteil beeinflussen werden.
+
+## osu!supporter erhalten
+
+![osu!store Produktbanner](img/store-product.jpg?1 "osu!supporter Produktbanner aus dem osu!store")
+
+Um osu!supporter zu erhalten, melde dich an (oder [registriere dich](/wiki/Registration)) und gehe dann auf die [Shop-Seite](https://osu.ppy.sh/store/products/supporter-tag). Verwende den Schieberegler oder die Monatsauswahl (unterhalb des Schiebereglers), um die Dauer des osu!supporter-Tags auszuwählen. Bei einer anderen Dauer ändert sich der Preis, den du zahlen musst. Die Währung ist der amerikanische Dollar (USD). Wenn dein Land eine andere Währung verwendet, benutze bitte einen Währungsrechner wie [OANDA](https://www.oanda.com/currency/converter/) oder [Google](https://www.google.com/search?q=usd+wechselkurs), um den Preis zu überprüfen. Die auf der osu!store-Seite angezeigten Preise gelten für jeden einzelnen Account.
+
+### Auswählen der Dauer
 
-## Einleitung
+!{Kauf von osu!supporter](img/selecting-duration.jpg?1 "Auswahl des Benutzers und der Dauer für osu!supporter")
```

Comment:
```suggestion
![Kauf von osu!supporter](img/selecting-duration.jpg?1 "Auswahl des Benutzers und der Dauer für osu!supporter")
```

Reply:
blame german keyboard for dumb bracket palcements :smile: